index.js

```diff
@@ -135,33 +147,35 @@ function exec(str, cols, opts) {
 
 	pre = rows.map(x => x.trim()).join('\n');
 
-	for (let j = 0; j < pre.length; j++) {
-		const y = pre[j];
+	for (const item of Array.from(pre).entries()) {
+		const i = item[0];
+		const x = item[1];
```

Comment:
Should we call `x` `char` or something (like in the `wrapWord` function)? A variable named `x` doesn't say much.